src/scene_battle_rpg2k.cpp

```diff
@@ -359,10 +359,16 @@ void Scene_Battle_Rpg2k::ProcessActions() {
 }
 
 bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase* action) {
+	/**
+     * Order of execution of BattleActionState:
+	 * ConditionHeal > Execute > Apply > (ResultPop > ResultPush) > Death > Finished.
+	 **/
```

Comment:
just doing a quick scroll ^^. This shouldn't be a doc-comment but two normal // comments.
But maybe before updating anything here I need to understand the new state stuff.